src/templates/powered_by_template.html

```diff
@@ -20,7 +20,7 @@ <h1>{{title}}</h1>
                 </thead>
                 <tbody class="list">
                     {% for key, value in data.items() %}
-                        {% if enable_query is none or value.enabled is enable_query %}
+                        {% if enable_query == none or value.enabled == enable_query %}
```

Comment:
Why this change?

Reply:
https://github.com/pallets/jinja/issues/520#issuecomment-164306945
`is` is a keyword--I was running into 500s when using the dev config and modifiying the query param; so went with `==`